playbooks/roles/deploy-osh/tasks/main.yml

```diff
@@ -130,17 +130,44 @@
     - horizon
     - run
 
+- name: Install openstack client
+  package:
+    name: python-openstackclient
+    state: present
+  register: _pkginstall
+  until: _pkginstall is success
+  retries: 5
+  delay: 2
+  tags:
+    - run
 
-# TODO(evrardjp): Use mechanism like glance to re-use component-install.yml code and allow any override.
-# TODO(evrardjp): Check stdout to ensure idempotency. In the meantime skip_ansible_lint
-- name: Setup openstack CLIs
-  command: ./tools/deployment/multinode/010-setup-client.sh
-  args:
-    executable: /bin/bash
+- name: Create /etc/openstack directory
+  file:
+    path: /etc/openstack
+    state: directory
+    owner: root
+    group: root
+    mode: 0755
+  tags:
+    - run
+
+- name: Save default clouds.yaml file
+  copy:
+    srt: clouds.yaml
+    dest: /etc/openstack/clouds.yaml
```

Comment:
why do we need to install that system wide rather than in the home directory?

Reply:
I guess we don't really _need_ it. This is just copying the functionality of helm's `010-setup-client.sh`

Reply:
That said, I think I could put it under `~/.config/openstack/` , unless there really is some reason to have it in global location.

Reply:
I think both are fine. Let's use home, it seems cleaner :)
We'll see if it eventually conflicts with the expectations of other things upstream too, and we could improve upstream then.

Reply:
OK